lib/parsers/entity_parser.js

```diff
@@ -60,7 +60,7 @@ function parse(args) {
     throw new Error('The JDL object and the database type are both mandatory.');
   }
   if (merged.applicationType !== GATEWAY) {
-    checkNoSQLModeling(merged.jdlObject, args.databaseType);
+    checkNoSQLModeling(args.baseName, merged.jdlObject, args.databaseType);
```

Comment:
Something bothers me:
  - I don't like having a function with more than 2 parameters (it could mean that said function has more than one responsibility for instance)
    - It can easily be fixed by passing an object with these parameters (`applicationName, jdlObject, applicationType`)